src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -707,6 +712,62 @@ public void renewalNoticeIsSentWhenPolicyDefinesRenewalNoticeConfiguration() {
     assertThatPublishedLogRecordEventsAreValid();
   }
 
+  @Test
+  public void shouldNotChargeOverdueOnRenewalWhenItemAgedToLostAndRefundFeePeriodPast() {
+
+    IndividualResource overDueFinePolicy = overdueFinePoliciesFixture.facultyStandard();
+    IndividualResource lostItemPolicy = lostItemFeePoliciesFixture.ageToLostAfterOneWeek();
+
+    val result = ageToLostFixture.createLoanAgeToLostAndChargeFeesWithOverdues(lostItemPolicy, overDueFinePolicy);
+
+    IndividualResource item = result.getItem();
+    IndividualResource user = result.getUser();
+    
+    final DateTime renewalDate = now(UTC).plusWeeks(9);
+    mockClockManagerToReturnFixedDateTime(renewalDate);
+    
+    IndividualResource renewedLoan =
+      loansFixture.overrideRenewalByBarcode(item, user,
+        OVERRIDE_COMMENT, null);
+
+    assertThat(renewedLoan, hasNoOverdueFine());
+  }
+
+  public void shouldNotChargeOverdueOnRenewalWhenItemDeclaredLostAndRefundFeePeriodPassed() {
+    ItemResource item = itemsFixture.basedUponSmallAngryPlanet();
+    UserResource user = usersFixture.jessica();
+
+    IndividualResource overduePolicy = overdueFinePoliciesFixture.facultyStandard();
+    IndividualResource lostItemPolicy = lostItemFeePoliciesFixture.ageToLostAfterOneWeek();
+
+    policiesActivation.use(PoliciesToActivate.builder()
+      .lostItemPolicy(lostItemPolicy)
+      .overduePolicy(overduePolicy)
+    );
+
+    IndividualResource loan = checkOutFixture
+      .checkOutByBarcode(item, user);
```

Comment:
Why is this split across two lines?

Reply:
At one point this statement had a lot more lines and was broken up so that I could better read it.  When I rewrote it, I neglected to reposition.  I'll fix it.